src/Compilers/Core/Portable/InternalUtilities/FatalError.cs

```diff
@@ -72,6 +72,9 @@ public static void OverwriteHandler(ErrorReporterHandler? value)
         /// This file is in linked into multiple layers, but we want to ensure that all layers have the same copy.
         /// This lets us copy the handler in this instance into the same in another instance.
         /// </remarks>
+#if NET6_0_OR_GREATER
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCode("Gets type by name")]
```

Comment:
Why didn't you follow the `TrimWarnings.` constant pattern here?

Reply:
This is internal, so I didn't feel like it mattered and it was a little nicer to see the string directly in code.